pkg/helper/pipeline/utils.go

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	"fmt"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//LastPipelineRun returns the last run for a given pipeline
+func LastPipelineRun(cs *cli.Clients, pipeline, ns string) (*v1alpha1.PipelineRun, error) {
```

Comment:
Could be name `LastRun` as the package is `pipeline`, so it would give `pipeline.LastRun`

Reply:
Also, we are only using `cs.Tekton`, so we should only pass this as argument (this would remove the dependency of this package to the `cli` package :angel: — which makes it more reusable as is)

Reply:
Done